modules/swagger-codegen/src/main/resources/swift3/APIs.mustache

```diff
@@ -42,6 +42,12 @@ open class RequestBuilder<T> {
 
     required public init(method: String, URLString: String, parameters: Any?, isBody: Bool, headers: [String:String] = [:]) {
         self.method = method
+
+        // This fixes a problem we have discovered with Apple's URL Encoding where plus characters are not URL encoded
+        // and end up as spaces on the server. See here: https://developer.apple.com/documentation/foundation/nsurlcomponents/1407752-queryitems
+        // The following ensures that plus signs are also encoded.
+        let allowedCharacters = CharacterSet(charactersIn: "+").inverted
```

Comment:
Allows everything but plus (bear in mind that the URL has already been encoded at this point).

Reply:
I assume that the encoding before this hasn't converted anything (e.g spaces) to pluses? Because that could be bad ..

Reply:
Test this, and not an issue. Spaces are encoded as %20